similar_dev_search/cli/get_similar_devs.py

```diff
@@ -0,0 +1,32 @@
+import click
+from prettytable import PrettyTable
+
+from similar_dev_search.constants import TEMP_PATH
+from similar_dev_search.services.file_system import JsonService
+from similar_dev_search.services.user_vectors import UserVectorService
+
+
+@click.command()
+@click.option('--dev_name', default='./', help='Developer name to search similar developers')
+def start_search(dev_name: str) -> None:
+    dataframe = fetch_data()
+    devs = UserVectorService().get_similar_dev(dataframe, dev_name)
+    table = PrettyTable(['Name', 'Probability'])
+    table.align = "r"
+    for dev in devs:
+        table.add_row([dev[0], dev[1]])
+    print(table)
+
+
+def fetch_data():
+    paths = JsonService.absolute_file_paths(TEMP_PATH + "jsons/")
```

Comment:
it should an argument to function with default value `TEMP_PATH ` - so you can configure it